### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by StefanJum**
* Update README.md
* Change mount tag from `rootfs` to `fs0`, since `fs0` is used by default by qemu-guest

### CODE REVIEW

**Comment by razvand** [comment]
@StefanJum , to make this compatible to other PRs, please remove the `Makefile` and `Makefile.uk`. It's simplest to remove the two commits. You should end up only with one commit for updating `README.md` and `kraft.yaml`. You can remove the Makefile commits and squash the other two commits into one.

**Review by gabrielmocanu — APPROVED** [review]
Looks good to me.

Reviewed-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

File: README.md [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
+make menuconfig
+```
+
+Build the application:
+```
+make
+```
+
+Run the application:
+- Create a sub-directory named fs0 in the application directory and extract the contents of minrootfs.tgz there.
```
Comment (razvand) [inline README.md]:
```suggestion
- Create a sub-directory named `fs0/` in the application directory and extract the contents of `minrootfs.tgz` there.
```

**Review by razvand — COMMENTED** [review]
I suggested some cosmetic changes.

**Review by razvand — APPROVED** [review]
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Pull request closed without merging**